src/tox/config/__init__.py

```diff
@@ -1045,13 +1048,22 @@ def make_hashseed():
     return str(random.randint(1, max_seed))
 
 
+class SkipThisIni(Exception):
```

Comment:
I'd probably derive this from ValueError as the file is present but we're unhappy with its content, rather than directly from the exception.

Reply:
Actually, I wanted to use `ValueError` directly, but there is another `ValueError` raise by this `__init__` a bit bellow and I didn't want to interfere with it.

Reply:
Oh, I thought this conversation is still going on. All good?

Reply:
Was a slight suggestion, not deal-breaker.

Reply:
Understood.